app/models/micropost.rb

```diff
@@ -0,0 +1,3 @@
+class Micropost < ApplicationRecord
+  scope :recent, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent, -> {order(created_at: :desc)}
```
Thêm 1 space trước {} e nhé